shared/postgres/scripts/init.sql

```diff
@@ -0,0 +1,12 @@
+DROP TABLE IF EXISTS employees;
+DROP SEQUENCE IF EXISTS employees_id_seq;
+CREATE TABLE employees (
```

Comment:
I'd rename the file to `create_table_employees.sql`, unless it _has_ to be named `init.sql`. Also the `insert_data.sh` would be more descriptive if it were `insert_employees.sh` IMO.